prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -166,10 +166,11 @@ class ReportFunction(
         // allow duplicates 'override' param
         val allowDuplicatesParam = request.queryParameters.getOrDefault(ALLOW_DUPLICATES_PARAMETER, null)
         val verbose = verboseParam.equals(VERBOSE_TRUE, true)
-        val report = try {
-            val optionsText = request.queryParameters.getOrDefault(OPTION_PARAMETER, "None")
-            val options = Options.valueOf(optionsText)
 
+        val optionsText = request.queryParameters.getOrDefault(OPTION_PARAMETER, "None")
+        val options = Options.valueOf(optionsText)
```

Comment:
I think this throws an error if `optionsText` isn't valid so it may need to be moved back within the try.

Reply:
@whytheplatypus fixed